libsolidity/lsp/FileRepository.cpp

```diff
@@ -51,6 +67,8 @@ string FileRepository::sourceUnitNameToUri(string const& _sourceUnitName) const
 		return "file:///" + _sourceUnitName;
 	else if (_sourceUnitName.find("/") == 0)
 		return "file://" + _sourceUnitName;
+	else if (auto const resolvedPath = tryResolvePath(_sourceUnitName); resolvedPath.message().empty())
```

Comment:
The actual condition in such ifs is really easy to miss. I'd split that in two lines (or just avoid this syntax):
```suggestion
	else if (
		auto const resolvedPath = tryResolvePath(_sourceUnitName);
		resolvedPath.message().empty()
	)
```

Reply:
We have much longer lines than that. I don't think that's particularly unreadable. But this is subjective. I'm not sure.

Reply:
For me it's not about the line length but about the condition blending in with the variable declaration. In most cases in our code such `if`s do not have the condition part after `;` and the condition is the variable. If it's not then IMO it's very easy to miss (I caught myself on that many times) and I think the condition should always be on a line of its own to make it more prominent.